test/e2e/mpi_job_test.go

```diff
@@ -164,8 +166,46 @@ var _ = ginkgo.Describe("MPIJob", func() {
 				mpiJob := createJobAndWaitForCompletion(mpiJob)
 				expectConditionToBeTrue(mpiJob, kubeflow.JobSucceeded)
 			})
-		})
 
+			ginkgo.It("should not be updated when managed externaly, only created", func() {
+				mpiJob.Spec.RunPolicy.ManagedBy = ptr.To(kubeflow.MultiKueueController)
+				ctx := context.Background()
+				mpiJob = createJob(ctx, mpiJob)
+
+				gomega.Consistently(func() error {
```

Comment:
I don't think we need to retry the Get with Consistently. I think waiting 1s before the Get should be enough. If ManagedBy wasn't respected then the Get would return updated object.